packages/babel-plugin-proposal-decorators/test/fixtures/2023-11-classes/inheritance/output.js

```diff
@@ -1,20 +1,18 @@
-var _initClass, _classDecs, _initClass2, _classDecs2, _Bar2;
+var _initClass, _initClass2, _Bar2;
 const dec = () => {};
-_classDecs = [dec1];
```

Comment:
The class decorators should be evaluated outside to ensure that the private environment is correct.

Reply:
Do you have an example?

Reply:
Oh I see
```js
let o;

class A {
  #x;
  foo() {
    @(o.#x)
    class B {
      @dec(() => o.#x) #x
    }
  }
}
```